### PR TITLE
Minor code cleanup

### PR DESCRIPTION
**Pull request by Boreeas**
Noticed while writing javadoc

* Block in BasicCubeGenerator has no effect
* Static Method in CubePrimer moved below non-static methods

### CODE REVIEW

File: src/main/java/cubicchunks/worldgen/generator/BasicCubeGenerator.java [hunk 1]
```diff
@@ -52,11 +52,6 @@ public void generateColumn(Column column) {
 				Coords.cubeToMinBlock(column.getX()),
 				Coords.cubeToMinBlock(column.getZ()),
 				Cube.SIZE, Cube.SIZE);
-
-		byte[] abyte = column.getBiomeArray();
-		for (int i = 0; i < abyte.length; ++i) {
-			abyte[i] = (byte) Biome.getIdForBiome(this.columnBiomes[i]);
```
Comment (Barteks2x) [inline src/main/java/cubicchunks/worldgen/generator/BasicCubeGenerator.java]: This is supposed to generate biomes, why did you remove that?

**Comment by Barteks2x** [comment]
And why make PR for such small changes? Why not clean up more and submit a bigger PR?

File: src/main/java/cubicchunks/worldgen/generator/BasicCubeGenerator.java (hunk 1 above)
Comment (Boreeas) [inline src/main/java/cubicchunks/worldgen/generator/BasicCubeGenerator.java]: There is nothing actually happening here: 
* `abyte` is never used
* `getIdForBiome` is only a map lookup

**Comment by Boreeas** [comment]
Because I'm actually working on cleaning up javadoc, and I didn't want to hide code changes in javadoc changes

**Comment by Barteks2x** [comment]
So maybe you could wait until you clean up everything you want and submit all code changes in one PR? That would avoid a lot of small cleanup PRs (unless there are so many changes that it will crash some browsers).

File: src/main/java/cubicchunks/worldgen/generator/BasicCubeGenerator.java (hunk 1 above)
Comment (Boreeas) [inline src/main/java/cubicchunks/worldgen/generator/BasicCubeGenerator.java]: Ah, I see what I missed. That's a nasty way to set biome ids though

Comment (Barteks2x) [inline src/main/java/cubicchunks/worldgen/generator/BasicCubeGenerator.java]: `Chunk.getBiomeArray` returns the *internal* biome array and if you modify that array - you modify biomes in that chunk.

**Comment by Boreeas** [comment]
I figured earlier/smaller is better, since it reduces the risk of merge conflicts and allows people to rebase their work quickly

**Comment by Barteks2x** [comment]
Smaller is better, up to a point. I could still merge it but it doesn't seem like the biome change is correct, and the second one is just reordering methods.
